projects/ccf-eui/src/app/shared/components/spatial-search-ui/spatial-search-ui.component.ts

```diff
@@ -81,4 +91,25 @@ export class SpatialSearchUiComponent {
 
   /** Emits when a node in the scene is clicked */
   @Output() readonly nodeClicked = new EventEmitter<SpatialSceneNode>();
+
+  /** Slider component */
+  @ViewChild('slider', { static: false }) slider: MatSlider;
+
+  /**
+   * Creates an instance of visibility menu component.
+   *
+   * @param el document element
+   * @param ga Analytics service
+   */
+  constructor(el: ElementRef<Element>) {
+    el.nativeElement.addEventListener('mousedown', () => {
```

Comment:
valueChange doesn't emit while I drag the slider so the values don't update until I let go. Think this got changed in the previous Angular update but we were able to use the legacy slider components until now. 

This was how I could get it to match the previous behavior. I realized I can use dragStart to simplify the code a bit, so that might help a little.